controller/node_controller.go

```diff
@@ -240,6 +288,12 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 		return err
 	}
 
+	// get settings of StorageOverProvisioningPercentage and StorageMinimalAvailablePercentage
+	overProvisioningPercentage, minimalAvailablePercentage, err := nc.ds.GetSettingsForDisk()
```

Comment:
It's too easy to mess up with returning two identical types of variables. It's better to either pass in as a typed struct or separate them into two functions. I think two functions are better here.

We can replace the datastore function with `GetSettingAsInt()`. It can check if the type is indeed `int` before return.